### PR TITLE
Binary Exponentiation in dart

### LINKED ISSUE #5618: implement Binary Exponent calculation in Dart
## 💥 Proposal

implement Binary Exponent calculation in Dart

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
 YES

### PR DESCRIPTION
**Pull request by ashuvssut**
### Description

implement binary-exponent in dart

## Related Issues or Pull Requests

closes #5618

### CODE REVIEW

File: Dart/maths/Binary_Exponentiation.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Dart Program to find Binary Exponent Iteratively and Recursively.
```
Comment (ankitaggarwal23) [inline Dart/maths/Binary_Exponentiation.dart]: - Add Multi-Line Comments 
- Add a beginner-friendly description at the top

File: Dart/maths/Binary_Exponentiation.dart [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+/**
+Sample Input and Output :
+
+Lets calculate Binary Exponent a^b
+Enter a
+3
+Enter b
+5
+243
+243
```
Comment (ankitaggarwal23) [inline Dart/maths/Binary_Exponentiation.dart]: Use proper display message for displaying outputs

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by ankitaggarwal23** [comment]
Add a repl.it link as well

**Comment by ashuvssut** [comment]
https://replit.com/join/cwzgewtw-ashutoshkhandua

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**